src/viur/core/skeleton.py

```diff
@@ -430,6 +434,11 @@ def ensure_is_cloned(self):
 
         return self
 
+    def readonly(self):
+        skel = self.ensure_is_cloned()
```

Comment:
Why does this skeleton to be cloned? It does not change the bone map or any other attribute. IMO we can set the attribute on any `SkeletonInstance`.